industrial_ci/test/example_ros.test

```diff
@@ -1,13 +0,0 @@
-<launch>
```

Comment:
File is used in https://github.com/ros-industrial/industrial_ci/blob/master/industrial_ci/CMakeLists.txt#L9